bin/pygrb/pycbc_grb_trig_cluster

```diff
@@ -187,10 +185,10 @@ with h5py.File(args.trig_file, "r") as h5f:
 # empty file (no triggers), so just copy the file
 if not time.size:
     shutil.copyfile(args.trig_file, outfile)
+    err_msg = "trigger file is empty\ncopied input file to {}".format(outfile)
     if args.verbose:
-        print("trigger file is empty")
-        print("copied input file to {}".format(outfile))
-    sys.exit(0)
+        print(err_msg)
+    raise RuntimeError(err_msg)
```

Comment:
Looking at this again now, I realized that this is not an error condition, is it? In that case you should not raise an exception, otherwise HTCondor or whatever will consider the job as failed. I am not sure how frowned upon `sys.exit(0)` is, but it does look like a practical shortcut in this case… Maybe add a FIXME to be improved later?

Reply:
And please do replace the if+`print()` with just a `logging.info()`.

Reply:
True, it is not an error.   Avoiding the `sys.exit(0)` could be achieved by placing all the lines of code that follow in an `else`.  That is kind of ugly, though.